docs/manual.rst

```diff
@@ -674,7 +674,7 @@ options. The exact mechanism is explained above the config options in the
 
 .. code:: python
 
-    # POSTS and PAGES contains (wildcard, destination, template) tuples.
+    # POSTS and PAGES contain (wildcard, destination, template) tuples.
```

Comment:
If you’re making changes to output samples, the changes should also be applied to the codebase.